### DISCUSSION TITLE
We started implementing the Renovate using the documentation which product team shared with us, As per documentation we are required to set a PAT as token in our config.js file. But we don't have that file, could you please provide your suggestions or any alternatives?

### DISCUSSION THREAD

**Discussion by Avinav5** [renovatebot/renovate#27410] [Request Help]
### How are you running Renovate?

None

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

We started implementing the Renovate using the [documentation](https://docs.renovatebot.com/modules/platform/azure/) which product team shared with us, As per documentation we are required to set a PAT as token in our config.js file. But we don't have that file, could you please provide your suggestions or any alternatives?
It's clear to us, Could you please help us with step-by-step of implementing Renovate on our Azure DevOps Repository?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
Read https://docs.renovatebot.com/getting-started/running/#global-config
